cmd/podman/exists.go

```diff
@@ -44,7 +44,7 @@ var (
 	}
 
 	_containerExistsCommand = &cobra.Command{
-		Use:   "exists",
+		Use:   "exists CONTAINER",
 		Short: "Check if a container exists in local storage",
 		Long:  containerExistsDescription,
 		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
would you mind adding examples for all the exists commands?

Reply:
Not at all; I'll work on it tomorrow AM. Thanks for the suggestion.